source/getting-started/index.markdown

```diff
@@ -41,7 +41,7 @@ We will need a few things to get started with installing Home Assistant. Links b
 
 1. Put the SD card in your SD card reader.
 1. Open Etcher, select the Hass.io image and flash it to the SD card.
-1. WiFi setup only: open the file `system-connections/resin-sample` (from the `resin-boot` volume on the SD card) with a text editor. Change `ssid` to be your network name and `psk` to be your password. Note that the Raspberry Pi 3 is a 2.4GHz WiFi device, so do not try to connect it to a 5GHz network.
```

Comment:
The note is not about the Raspberry Pi B+. Thus, it's still valid.

Reply:
The note is about "Raspberry Pi 3", which to me includes all Pi 3 models, including B and B+.

Reply:
ok. Then it should be changed to something 'Raspberry Pi 3B (standard version, not the +-version)'

Reply:
I think that's too much detail, and that just removing the note would be a better idea.

Reply:
Then I think we should just replace it with:
`Note: When connecting to a 5Ghz-only network, make sure your device supports that. If you don't know what that means, then you have probably nothing to worry about.`

Reply:
WFM

Reply:
Just to make it clear, because @NovapaX came up with the improved wording, I think it would be a good idea for them to submit a superseding PR to get the authorship logged right.

Reply:
Feel free to take make the change @scop.
Members of the Home Assistant project (like me, @fabaff and @NovapaX) make suggestions on a daily basis and are not seeking for the credit on every single change/PR 😉